packages/generator/generators/draft/templates/README.txt

```diff
@@ -0,0 +1,27 @@
+---
+title: "<%= componentName %>"
+navTitle: "<%= componentName %>"
+summaryParagraph: <%= componentName %>s are a neat component
+tags: [""]
+demoStoryId:  <%= componentName %>-react--default-kaizen-site-demo
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+<WhenToUseAndWhenNotToUse>
+
+<WhenToUse>
+* Use this for the thing
+</WhenToUse>
+
+<WhenNotToUse>
+* Don't use this for the other thing though! 
+</WhenNotToUse>
+</WhenToUseAndWhenNotToUse>
+
+## See also
+
```

Comment:
It's been useful to designers to see Markdown syntax examples. Perhaps show example link to commonly referenced component:

`* [Button](/components/button/)`